tailwind.config.js

```diff
@@ -0,0 +1,45 @@
+function withOpacityValue(variable) {
+  return ({ opacityValue }) => {
+    if (opacityValue === undefined) {
+      return `rgb(var(${variable}))`;
+    }
+    return `rgb(var(${variable}) / ${opacityValue})`;
+  };
+}
+
+module.exports = {
+  prefix: "tw-",
+  content: ["./src/**/*.{html,ts}"],
+  safelist: [
+    {
+      pattern: /tw-(border|bg)-(primary|secondary)/,
+    },
+    {
+      pattern: /tw-text-(primary|secondary)-btn-text/,
+    },
+    {
+      pattern: /tw-bg-(primary|secondary)/,
+      variants: ["hover"],
+    },
+  ],
+  theme: {
+    colors: {
+      primary: withOpacityValue("--color-primary"),
```

Comment:
I would prefer investigating if we could go towards using a color palette instead of hard-coding single-use variables like `primary-btn-hover`. Ideally we would be able to use `primary-500` for backgrounds, `primary-600` for borders, and maybe `primary-700` for hover state. Which should allow us to minimize the unique variables.

Reply:
Unique variables were requested so that it was easier to maintain and fix bugs after the initial release of dark mode. Now that most issues are fixed it should be quite simple to switch to this method for them instead